texfont calls pfaedit, but Debian has already fontforge, change the name
---
 scripts/context/perl/texfont.pl |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: context-2008.01.02/scripts/context/perl/texfont.pl
===================================================================
--- context-2008.01.02.orig/scripts/context/perl/texfont.pl	2006-12-13 18:25:07.000000000 +0100
+++ context-2008.01.02/scripts/context/perl/texfont.pl	2008-01-10 07:35:51.000000000 +0100
@@ -1078,8 +1078,8 @@
         report ("destination file : $pfbfont") ; }
     else
       { error ("missing otf/ttf source file") }
-    open (CONVERT, "| pfaedit -script -") || error ("couldn't open pipe to pfaedit") ;
-    report ("pre-processing with : pfaedit") ;
+    open (CONVERT, "| fontforge -script -") || error ("couldn't open pipe to fontforge") ;
+    report ("pre-processing with : fontforge") ;
     print CONVERT "Open('$infont');\n Generate('$pfbfont', '', 1) ;\n" ;
     close (CONVERT) }
 
